### DISCUSSION TITLE
Open GoPro Updates for Hero 10 1.30 Firmware Release

### DISCUSSION THREAD

**Discussion by tcamise-gpsw** [gopro/OpenGoPro#122] [Announcements]
Hello all! The following updates have been made as part of the Hero 10 1.30 Firmware release:
- Instructions for wired control added
    - change "WiFi" spec to "HTTP" spec and delineate WiFi / wired differences with the spec
- Webcam commands / statuses added for wired HTTP
- Align Get / Set date time across BLE and HTTP
     - both BLE and HTTP now have Get and Set Date commands
     - deprecated the DateTime status ID
- Added hilight tag functionality to HTTP
     - add / remove Hilight commands for images and videos
- Added a CSharp Webcam demo (for Windows)
- Updated Python SDK (see its changelog for more information)

**Comment by KonradIT** [comment]
USB control is so cool, glad it was released.
